### PR TITLE
Fixed launch.json file

### LINKED ISSUE #8989: Update VSCode launch configuration file in 'Debug your add-to-app module'
### Page URL

https://docs.flutter.dev/add-to-app/debugging/

### Page source

https://github.com/flutter/website/tree/main/src/add-to-app/debugging.md

### Describe the problem

Mistake in ```.vscode/launch.json``` formatting:
```
{
  name: "Flutter: Attach",
  request: "attach",
  type: "dart",
}
```

should be:
```
{
    // Use IntelliSense to learn about possible attributes.
    // Hover to view descriptions of existing attributes.
    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
    "version": "0.2.0",
    "configurations": [
        {
            "name": "Flutter: Attach",
            "request": "attach",
            "type": "dart",
        },
    ]
}
```

### Expected fix

Change the document

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by atsansone**
Fixes #8989

### CODE REVIEW

**Review by parlough — COMMENTED** [review]
Not sure what happened in this PR, but its history is messed up, and perhaps needs a rebase. Could you take another look @atsansone? Thanks!

File: src/_includes/docs/debug/debug-flow-ios.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ To generate the needed iOS platform dependencies,
 run the `flutter build` command.
 
 ```console
-flutter build ios --config-only --no-codesign --debug
+$ flutter build ios --config-only --no-codesign --debug
```
Comment (domesticmouse) [inline src/_includes/docs/debug/debug-flow-ios.md:7]: Should we include command output as well?

File: src/_includes/docs/debug/debug-flow-vscode-as-start.md [hunk 2]
```diff
@@ -33,4 +33,8 @@
    - **Launch in browser**: This button opens this page in the
      default browser of your device.
    - **Launch in app**: This button opens this page within your app.
-     This button only works for iOS or Android. Desktop apps launch a browser.
\ No newline at end of file
+     This button only works for iOS or Android. Desktop apps launch a browser.
+
+{% if include.add == 'launch' -%}
+{% include docs/debug/vscode-flutter-attach-json.md %}
+{% endif -%}
```
Comment (domesticmouse) [inline src/_includes/docs/debug/debug-flow-vscode-as-start.md:40]: Add NL at EOF

**Review by domesticmouse — APPROVED** [review]

File: src/_includes/docs/debug/debug-flow-ios.md (hunk 1 above)
Comment (atsansone) [reply src/_includes/docs/debug/debug-flow-ios.md:7]: @domesticmouse : It's in the next code block. I split them so readers can copy the command without the response.

**Review by sfshaza2 — APPROVED** [review]
lgtm

**Pull request merged**